### ISSUE TITLE
WPF and .net 5.0

### ISSUE THREAD

**Issue by Malavos** [yandexmobile/metrica-sample-dotnet#18]
Hey everyone,

Is there any plans to update the SDK for WPF? With the store supporting win32 apps, it would be awesome to count with Yandex.Metrica full tools to not only push events, but also have sessions recording and much more. I know the current nuget stopped being updated when MS dropped the WP and UWP support. But are there are any plans for this future with Win32 and .net 5.0?

**Comment by ghost** [comment]
Добрый день!Ваше письмо получено и зарегистрировано под уникальным номером № 14338613. Ответ обязательно будет предоставлен, пожалуйста, ожидайте.Просим Вас не отправлять повторное письмо по данному вопросу - оно будет считаться новой заявкой и может быть обработано позднее.Помощь сервиса: https://yandex.ru/support/direct/?from=email--С уважением,Отдел клиентского сервисател.: 8 800 234-24-80 (звонок из регионов России бесплатный)тел.: +7 495 739-37-77&nbsp13.10.2021, 21:55, "Malavos" ***@***.***>:
Hey everyone,
Is there any plans to update the SDK for WPF? With the store supporting win32 apps, it would be awesome to count with Yandex.Metrica full tools to not only push events, but also have sessions recording and much more. I know the current nuget stopped being updated when MS dropped the WP and UWP support. But are there are any plans for this future with Win32 and .net 5.0?

**Comment by alexeygritsenko** [comment]
Тогда уж добавьте сразу .net 6, пятая уходит в небытие не вижу смысла её поддерживать.

**Comment by Malavos** [comment]
@alexeygritsenko Yeah, sure. I was going to disagree with you, but I took a look at 5.0 and 6.0 support lifecycle, and saw that 6.0 will be on LTS. So yeah, maybe add support to 6.0. But would be great to have this defined officially, since we got a integration coming here, and we are thinking about using Yandex or gapps.
